packages/core/lib/OneBehindFetch.js

```diff
@@ -15,58 +15,46 @@
  */
 'use strict';
 
-const axios = require('axios');
+let fetch = require('node-fetch');
 const MaxAge = require('./MaxAge.js');
 
+const cache = new Map();
+
 /**
  * Implements fetch with a one-behind-caching strategy.
+ *
+ * @param {String|Request} input - path or Request instance
+ * @param {Object} init - fetch options
+ * @return Promise<Response>
  */
-class OneBehindFetch {
-  /**
-   * Creates a new OneBehindFetch.
-   *
-   * @returns {OneBehindFetch}
-   */
-  static create() {
-    return new OneBehindFetch(axios);
-  }
-
-  /**
-   * Creates a new OneBehindFetch.
-   *
-   * @param {Axios} axio request handler
-   * @returns {OneBehindFetch}
-   */
-  constructor(axios) {
-    this.cache_ = {};
-    this.axios_ = axios;
+async function oneBehindFetch(input, init) {
```

Comment:
This function is now with a mix of async / await and Promise syntax. I find this a bit confusing. Consider refactoring to settle on a single syntax. 

eg: One the last line, this could work: `return (await result).clone()`, but it may not be clear enough and breaking into something like the below may be clearer:

```
const response = await result;
return result.clone();
```

Reply:
done